opentelemetry/sidecar/otelcol.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: otel-collector-example
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: otel-collector-example
+spec:
+  mode: sidecar
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+      opencensus:
+      otlp:
+        protocols:
+          grpc:
+          http:
+      zipkin:
+    processors:
+      batch:
+        # Batching helps better compress the data and reduce the number of outgoing
```

Comment:
it actually does not do any compression as far as I know

Reply:
It's what the first line from the [README.md file from the `batchprocessor`](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md) documentation claims. Should I change it?